### PR TITLE
Initial docs for project issue processes

### PR DESCRIPTION
**Pull request by asmacdo**

### CODE REVIEW

File: website/content/en/docs/contribution-guidelines/issue-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Issue Lifecycle
+weight: 30
+---
+
+The Operator SDK project tracks bugs, feature requests, and questions
+with [GitHub
+issues](https://github.com/operator-framework/operator-sdk/issues).
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/issue-lifecycle.md:8]: You can keep your lines short by putting the urls at the bottom of the document. 
```
[GitHub issues][gh-issues]

...
[gh-issues](https://github.com/operator-framework/operator-sdk/issues)
```
We've done that in a number of places.

File: website/content/en/docs/contribution-guidelines/issue-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Triage
+
+Each week, there is a [community
+meeting](https://github.com/operator-framework/community/#operator-sdk-community-issue-triagegrooming)
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/issue-lifecycle.md:13]: use the advice from above regarding urls.

File: website/content/en/docs/contribution-guidelines/issue-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+to triage issues. Each issue that has been filed since the previous
+meeting is discussed, GitHub labels are applied, and the issue is added
+to a Milestone. Additionally, anyone can request that a previously
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/issue-lifecycle.md:16]: would it be helpful to link to `Milestone`?  https://github.com/operator-framework/operator-sdk/milestones

**Review by jmrodri — CHANGES_REQUESTED** [review]
Just a minor url change.

**Comment by jberkhahn** [comment]
seems fine
/lgtm

**Review by camilamacedo86 — APPROVED** [review]
It shows fine for me as well 👍 
/lgtm

**Pull request merged**